_studio/mfx_lib/cmrt_cross_platform/src/cmrt_cross_platform.cpp

```diff
@@ -30,9 +30,6 @@ const vm_char * DLL_NAME_LINUX = VM_STRING("igfxcmrt32.so");
 #else
 #error "undefined configuration"
 #endif
-#if defined(ANDROID)
-const vm_char * DLL_NAME_ANDROID = VM_STRING("libigfxcmrt.so");
-#endif
 
```

Comment:
Why this went away? @dmitrybrazhkin : FYI

Reply:
Hm, looks like some merge issue, will look.